File: python/job/handlers/____method____.py
"""____description____"""


def ____method____(params: dict, headers: dict) -> dict:
    """The queue lane: work nobody waits for.

    A payload arrives as untyped JSON, so validate it here and raise on anything
    else — raising fails the job, which is what makes it retryable.
    """
    print(f"____name____:____method____ received {params}", flush=True)
    return {"ok": True}

File: typescript/job/handlers/____method____.ts
import type { FunctionHandler } from '@constructive-functions/types';

export interface Result {
  ok: boolean;
}

/**
 * The queue lane: work nobody waits for. A payload arrives as untyped JSON, so
 * validate it at the boundary and fail loud on anything else — a malformed
 * payload is a caller bug, not an `undefined` three frames down. Throwing fails
 * the job, which is what makes it retryable.
 */
export const ____method____: FunctionHandler<Record<string, unknown>, Result> = async (
  params,
  context
) => {
  context.log.info('____name____:____method____ received', { params });
  return { ok: true };
};
